cmd/edit/cluster/cmd.go

```diff
@@ -463,6 +541,91 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  Registries Config *******/
+	updateRegistriesConfig := false
+	if insecureRegistries != nil || blockedRegistries != nil || allowedRegistriesForImport != nil ||
+		allowedRegistries != nil || platformAllowlist != "" || additionalTrustedCa != "" {
+		updateRegistriesConfig = true
+	}
```

Comment:
This could be simplified
```suggestion
	updateRegistriesConfig := insecureRegistries != nil || blockedRegistries != nil || allowedRegistriesForImport != nil ||	allowedRegistries != nil || platformAllowlist != "" || additionalTrustedCa != ""
```

Reply:
Refined the updateRegistriesConfig